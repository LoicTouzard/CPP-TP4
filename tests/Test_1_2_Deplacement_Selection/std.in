C c 0 0 1
R rect 0 2 2 0
S s -3 3 3 -3
MOVE s 2 2
LIST
EXIT
