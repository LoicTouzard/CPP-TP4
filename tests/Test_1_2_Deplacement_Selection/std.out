OK
#New object :c
OK
#New object :rect
OK
#New object : s
OK
C c 2 2 1
R rect 2 4 4 2
